Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Company.FunctionApp.csproj

```diff
@@ -15,7 +15,7 @@
   <!--#endif -->
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.20.1" />
```

Comment:
FYI Worker 1.21.0 will be released today(1/29).

Reply:
We can wait on that then if preferred.